setup.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import setuptools
+
+setuptools.setup(
+    name='beagle_cli',
+    version='20.1.0',
+    scripts=['beagle_cli'] ,
```

Comment:
Maybe we can call script differently then `beagle_cli`? beagle_cli is maybe two long? Any suggestions? @allanbolipata @nikhil @timosong

Reply:
how about `beaglecli`? I usually tab-complete through the names anyway